log.go

```diff
@@ -0,0 +1,46 @@
+package authenticated_block_store
```

Comment:
Same as other occurrence: package names should be a single word.

Reply:
I'd be happier if it were actually a word, rather than multiple stuck together. Consider package "blockstore" nested inside the "authenticated" folder?

Reply:
I've changed it to blockstore for now. We can change it as we wish.